### PR TITLE
add: ContactUs page(none responsive)

### PR DESCRIPTION
**Pull request by elenamkoian**

### CODE REVIEW

File: src/assets/contact-us-page-photos/1897.png
Comment (evosliza) [inline src/assets/contact-us-page-photos/1897.png:1]: let's rename this file to smth descriptive like gradient blur or darken or idk

File: src/assets/icons/arrowRight/arrowRight.svg
Comment (evosliza) [inline src/assets/icons/arrowRight/arrowRight.svg:1]: make sure this doesn't break the design in other usage places

File: src/pages/ContactUsPage/ContactUsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+};
+
+const ContactUsForm = () => {
+    const { register, handleSubmit, formState: { errors }, reset } = useForm<ContactUsFormInputs>({
+      defaultValues: formDefaultValues,
+    });
+
+    const onSubmit = (data: ContactUsFormInputs) => {
+      reset(formDefaultValues);
+      console.log("Form submitted:", data);
```
Comment (menuason) [inline src/pages/ContactUsPage/ContactUsForm.tsx]: do we need this console.log?

File: src/pages/ContactUsPage/ContactUsForm.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    };
+
+    return (
+      <div className="flex flex-col mr-[104px] gap-8 ">
```
Comment (menuason) [inline src/pages/ContactUsPage/ContactUsForm.tsx:30]: is it normal this "mr-104px"?

File: src/pages/ContactUsPage/ContactUsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <img
+          src="src/assets/contact-us-page-photos/1897.png"
+          className="absolute top-0 left-0 right-0 bottom-0"
+          alt="Bloom House"
+        />
+        <div
+          className="absolute top-0 left-0 right-0 bottom-0 w-[886px] text-center text-white flex flex-col gap-2 m-auto mt-20">
+          <h2 className="lg:text-4xl font-medium">Contact Us</h2>
+          <div className="flex items-center gap-3 justify-center font-extralight cursor-pointer">
+            Home
```
Comment (menuason) [inline src/pages/ContactUsPage/ContactUsPage.tsx]: i think we need to have a link for home

File: src/pages/ContactUsPage/ContactUsForm.tsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
+import { useForm } from "react-hook-form";
```
Comment (evosliza) [inline src/pages/ContactUsPage/ContactUsForm.tsx:1]: please format this file

**Pull request merged**